infra/tf/k8s_infra/traefik.tf

```diff
@@ -66,9 +66,13 @@ resource "helm_release" "traefik" {
 		additionalArguments = [
 			"--providers.http.endpoint=http://rivet-api-route.rivet-service.svc.cluster.local/traefik/config/core?token=${module.traefik_secrets.values["rivet/api_route/token"]}",
 			"--providers.http.pollInterval=2.5s",
-			# See docs/infrastructure/API_TIMEOUTS.md
+			# See docs/infrastructure/TIMEOUTS.md
```

Comment:
Can we add a lower timeout on the ATS service specifically? Not sure how this converts to the CRD: https://doc.traefik.io/traefik/routing/services/#forwardingtimeouts